tests/factories.py

```diff
@@ -0,0 +1,24 @@
+from importlib import import_module
+
+from django.conf import settings
+
+import factory
+import factory.fuzzy
+from sessionprofile.models import SessionProfile
+
+
+class SessionProfileFactory(factory.django.DjangoModelFactory):
+
+    session_key = factory.fuzzy.FuzzyText(length=40)
+
+    class Meta:
+        model = SessionProfile
+
+    @classmethod
+    def create(cls, **kwargs):
```

Comment:
Kinda hack but it works. Might have the same issue as admin if not refectched

Reply:
Would be nice to make use of [one of these ](https://factoryboy.readthedocs.io/en/stable/reference.html#post-generation-hooks)methods for this

Reply:
De-hackified it